syntax                  = "proto3";

/**********通用消息体********/

message CommonMsg 
{
  string version         = 1;
  string type           = 2;
  int32 encrypt         = 3;
  int32 compress        = 4;
  bytes data            = 5;
}
